src/lib.rs

```diff
@@ -130,11 +126,13 @@ impl AtomicF64 {
 ///
 #[track_caller]
 #[inline(always)]
-pub(crate) fn assert_valid_sample_rate(sample_rate: SampleRate) {
-    if sample_rate.0 == 0 {
+pub(crate) fn assert_valid_sample_rate(sample_rate: f32) {
+    // 1000 Hertz is a just a random cutoff, but it helps a if someone accidentally puts a
+    // timestamp in the sample_rate variable
+    if sample_rate <= 1000. {
```

Comment:
We could eventually pick values from the spec here : https://webaudio.github.io/web-audio-api/#dom-baseaudiocontext-createbuffer-samplerate